src/py4dgeo/epoch.py

```diff
@@ -367,9 +389,14 @@ def load(filename):
                 cloudfile = zf.extract("cloud.laz", path=tmp_dir)
                 lasfile = laspy.read(cloudfile)
                 cloud = np.vstack((lasfile.x, lasfile.y, lasfile.z)).transpose()
-
+                try:
+                    normals = np.vstack(
+                        (lasfile.NormalX, lasfile.NormalY, lasfile.NormalZ)
+                    ).transpose()
+                except:
```

Comment:
What exception type are you expecting here? Find it out and make it explicit, otherwise you are silently ignoring any errors that might occur in the `try` block. Even better, find a condition for an `if` statement to determine whether the file stores normals or not and use that instead of a `try`/`except` block.